lib/dogapi/v1/monitor.rb

```diff
@@ -17,9 +17,13 @@ def monitor(type, query, options = {})
       end
 
       def update_monitor(monitor_id, query, options)
-        body = {
-          'query' => query,
-        }.merge options
+        body = {}.merge options
+        unless query.nil?
+          body = {
+            'query' => query
+          }.merge body
+          warn '[DEPRECATION] query param is no longer required for update'
```

Comment:
I would not warn here. If someone does want to actually update the query, they should not see a deprecation message.

Reply:
👍

Reply:
Actually you were right to have this message. I misunderstood how all of it worked.

Reply:
Ok just put it back in with the updates!